renku/models/provenance/activities.py

```diff
@@ -79,6 +80,10 @@ class Activity(CommitMixin):
             '@reverse': 'prov:activity',
         }, kw_only=True, hash=False
     )
+    influenced = jsonld.ib(
+        context='prov:influenced',
```

Comment:
where is this used?

Reply:
It's needed to mark parent directories as changed.